proposals/Exceptions.md

```diff
@@ -280,6 +280,30 @@ calls until a corresponding, enclosing try block is found. It may also associate
 a stack trace that can be used to report uncaught exceptions. However, the
 details of this is left to the embedder.
 
+### Traps and JS API
+
+The `catch` instruction catches exceptions generated by the `throw` instruction,
+but is not supposed to catch traps. The rationale for this is traps, which
+corresponds to hardware failures in real machines, are not generally needed to
```

Comment:
it's not always hardware failures; I think it's more generally that the failures are not locally recoverable.

Reply:
Fixed to "The rationale for this is that in general traps are    
not locally recoverable and are not needed to be handled in locals scopes like   
try-catch.".